tensorflow/python/ops/image_ops_impl.py

```diff
@@ -454,6 +454,19 @@ def flip_left_right(image):
   Outputs the contents of `image` flipped along the width dimension.
 
   See also `reverse()`.
+    
```

Comment:
Oh, before rebasing on master, can you also make sure there is no trailing whitespace?